Tests/TestData/greeksindicator/american/third_party_1_greeks.csv

```diff
@@ -0,0 +1,19 @@
+,,,,model_price,model_price,model_impliedVol,model_impliedVol,model_delta,model_delta,model_gamma,model_gamma,model_vega,model_vega,model_theta,model_theta,model_undPrice,TTM,interest_rate,dividend_yield
+right,,,,-1,1,-1,1,-1,1,-1,1,-1,1,-1,1,,,,
+ticker,expiry,strike,time,,,,,,,,,,,,,,,,
```

Comment:
I don't understand these headers lines? which seem to be skip while reading? Same for the file bellow

Reply:
These are files produced by python pandas when we're caching the raw data. Since the raw dataframe have 2-level columns, so it is displayed like this. Read the csv by pandas can reproduce that 2-level column dataframe